packages/website/pages/index.js

```diff
@@ -102,14 +102,23 @@ function About() {
         URL (<code>ipfs://&lt;cid&gt;</code>). Use this IPFS URL in your NFT
         data to refer to <strong>off-chain</strong> data (e.g., the metadata
         field in your NFT, the image field in your metadata) as a pointer to the
-        content.
+        content itself, so no one can dispute what your NFT is.
       </p>
       <p className="lh-copy">
         Filecoin provides long-term storage for the data ensuring that even if
```

Comment:
**suggestion (non-blocking):** Comma after data as below.

"Filecoin provides long-term storage for the data, ensuring that even if"